src/nonconsensus/outside_hash_image/outside_hash_image.ml

```diff
@@ -0,0 +1 @@
+src/lib/outside_hash_image/outside_hash_image.ml
```

Comment:
I think this symlink needs to be relative: `../../lib/outside_hash_image/outside_hash_image.ml`

Reply:
thanks will give it a go